ci/travis_script_java.sh

```diff
@@ -22,8 +22,8 @@ set -e
 JAVA_DIR=${TRAVIS_BUILD_DIR}/java
 pushd $JAVA_DIR
 
-export MAVEN_OPTS="$MAVEN_OPTS -Dorg.slf4j.simpleLogger.defaultLogLevel=warn"
-if [ $ARROW_TRAVIS_JAVA_BUILD_ONLY == "1" ]; then
+export MAVEN_OPTS="$MAVEN_OPTS -Dorg.slf4j.simpleLogger.defaultLogLevel=info"
```

Comment:
Why change the log level here?

Reply:
without this, the output was too concise. We can't figure out for example if it ran "memory" tests or not. Or, how many tests ran.

https://travis-ci.org/apache/arrow/jobs/482786298

Reply:
Yeah, it's probably best to have some output and hopefully with these changes it will be much smaller and not interfere.

How about just removing these exports then since `info` is the default maven log level anyway?